assets/js/modules/pagespeed-insights/components/dashboard/DashboardPageSpeed.test.js

```diff
@@ -193,6 +194,45 @@ describe( 'DashboardPageSpeed', () => {
 		expect( getByLabelText( /mobile/i ) ).not.toHaveClass( activeClass );
 	} );
 
+	it( 'displays refreshing states when the `Run test again` button is clicked', async () => {
+		freezeFetch(
+			/^\/google-site-kit\/v1\/modules\/pagespeed-insights\/data\/pagespeed/
+		);
+		// needs second freezeFetch call, as one is for desktop and the other for mobile
```

Comment:
Single line comments should start from a capital letter and end with a full stop:
```suggestion
		// Needs second freezeFetch call, as one is for desktop and the other for mobile.
```